SingleCellPatch/generate_trajectories.py

```diff
@@ -398,30 +399,37 @@ def process_site_build_trajectory(site_supp_files_folder):
     # Mapping to centroid positions
     cell_positions_dict = {k: dict(cell_positions[k][0] + cell_positions[k][1] + cell_positions[k][2]) for k in cell_positions}
     # Mapping to size of segmentation
-    intensities_dict = {}
+    cell_size_dict = {}
```

Comment:
Thanks for the renaming!

Reply:
Sure! I rename some variables based on my understanding of the code, but let me know if some of the changes don't make sense.